### PR TITLE
Corrected typos.

### PR DESCRIPTION
**Pull request by tkuchida**
Used https://github.com/EWSoftware/VSSpellChecker/

### CODE REVIEW

**Comment by chrisdembia** [comment]
Woohoo! I'll read through the changes tonight.

File: OpenSim/Actuators/Millard2012AccelerationMuscle.cpp [hunk 1]
```diff
@@ -138,7 +138,7 @@ void Millard2012AccelerationMuscle::constructProperties()
     constructProperty_fiber_force_length_damping(1e-2);
     constructProperty_fiber_compressive_force_cos_pennation_damping(1.0);
 
-    //Linear fiber damping as in Shutte's model
+    //Linear fiber damping as in Schutte's model
```
Comment (chrisdembia) [inline OpenSim/Actuators/Millard2012AccelerationMuscle.cpp]: Impressive VS spell checker!! It knows the biomechanics literature.

**Comment by chrisdembia** [comment]
I'd say we should run the spellchecker on appveyor, if there was an easy way to add words to its dictionary (is there?)

**Comment by chrisdembia** [comment]
Looks like we can: https://github.com/EWSoftware/VSSpellChecker/wiki/af34b863-6a1c-41ed-bcf2-48a714686519#user-dictionary.

@tkuchida was it easy to set up and use? Do you think we should use it in appveyor?

File: OpenSim/Actuators/Millard2012AccelerationMuscle.cpp (hunk 1 above)
Comment (sherm1) [inline OpenSim/Actuators/Millard2012AccelerationMuscle.cpp]: > Impressive VS spell checker!! It knows the biomechanics literature.

I bet it got some help.

**Comment by tkuchida** [comment]
> was it easy to set up and use?

It was easy to set up. In VS2015, go to Tools -> Extensions and Updates…, click "Online" and search for "Visual Studio Spell Checker". The only configuration I did was set "Ignore words in all uppercase" to "No".

> Do you think we should use it in appveyor?

It wasn't a completely automated process, so I don't know how Appveyorable it is. There are lots of comments that refer to variable names that aren't English words. My suggestion is for VS users to install the extension and watch out for the pink underlines.

File: OpenSim/Simulation/Manager/Manager.h [hunk 2]
```diff
@@ -71,7 +71,7 @@ class OSIMSIMULATION_API Manager
     Storage *_stateStore;
 
    int _steps;
-   /** Number of integration step trys. */
+   /** Number of integration step tries. */
```
Comment (chrisdembia) [inline OpenSim/Simulation/Manager/Manager.h]: This is inconsistent with the name of the corresponding variable but I think it's still an improvement.

File: OpenSim/Tools/InverseKinematicsTool.cpp [hunk 3]
```diff
@@ -542,7 +542,7 @@ void InverseKinematicsTool::updateFromXMLNode(SimTK::Xml::Element& aNode, int ve
                     aNode = updDocument()->getRootDataElement();
                 }
                 else
-                ;   // Somthing wrong! bail out
+                ;   // Something wrong! bail out
```
Comment (chrisdembia) [inline OpenSim/Tools/InverseKinematicsTool.cpp]: that's kinda confusing (an else clause with nothing)

**Comment by chrisdembia** [comment]
This is awesome!

File: OpenSim/Tools/InverseKinematicsTool.cpp (hunk 3 above)
Comment (tkuchida) [inline OpenSim/Tools/InverseKinematicsTool.cpp]: > that's kinda confusing (an else clause with nothing)

I agree. I guess we can either change it or.

Comment (chrisdembia) [inline OpenSim/Tools/InverseKinematicsTool.cpp]: hahahahah it took me a while to get it.

**Pull request merged**